### PR TITLE
Daos 4263 test: evt_ctl and btree cleanup

### PR DESCRIPTION
**Pull request by marcelarosalesj**

### CODE REVIEW

File: src/vos/tests/evt_ctl.c [hunk 1]
```diff
@@ -2435,17 +2344,34 @@ main(int argc, char **argv)
...
+	for(j=0; j<argc;j++){
+		if(strcmp(argv[j],"-t") == 0 ) {
+			rc = run_internal_tests();
+			if (rc != 0)
+				D_PRINT("Internal tests failed with rc="DF_RC"\n", DP_RC(rc));
+		}
+	}
+	
+	/* Execute the sequence of tests */
+	int start_idx;
```
Comment (jolivier23) [inline src/vos/tests/evt_ctl.c:2364]: For better or worse, we always put variables at the top of the block.   And generally, we align variable names with tabs (plus a space to align a pointer type with a non pointer type)....

int[tab]*x;
int[tab] y; /* note the space where the * would be. */

This isn't consistent throughout daos but it is pretty consistent in VOS

File: src/common/tests/btree.c [hunk 2]
```diff
@@ -994,6 +896,88 @@ static struct option btr_ops[] = {
...
+}
+
+static int
+run_cmd_line_test(char* test_name, char** args, int start_idx, int stop_idx)
+{
+	test_group_args = args;
+	test_group_start = start_idx;
+	test_group_stop = stop_idx;
+
+	const struct CMUnitTest btree_test[] = {
```
Comment (jolivier23) [inline src/common/tests/btree.c:975]: similar comment to below, variables should be at top of code blocks according to our conventions.

File: src/vos/tests/evt_ctl.c (hunk 1 above)
Comment (marcelarosalesj) [reply src/vos/tests/evt_ctl.c:2364]: Okay, I'll send another PR.
Thanks Jeff!

**Comment by marcelarosalesj** [comment]
I'm closing this PR because it was sent from a branch in my fork. I sent another one from a brach in daos-stack/daos repository

**Pull request closed without merging**